src/renderer/views/Watch/Watch.js

```diff
@@ -339,6 +345,22 @@ export default defineComponent({
       this.resetAutoplayInterruptionTimeout()
     },
 
+    setViewingModeOnFirstLoad: function () {
+      switch (this.defaultViewingMode) {
+        case 'theatre':
+          this.useTheatreMode = this.theatrePossible
+          return
+        case 'fullscreen':
+          this.startNextVideoInFullscreen = true
+          return
+        case 'fullwindow':
+          this.startNextVideoInFullwindow = true
+          return
+        case 'pip':
+          this.startNextVideoInPip = true
```

Comment:
Could you please use `break` instead of `return` here, as you aren't using it to return specific values.

Reply:
This was my way of making sure that this event listener would fire properly. I can change this if there's a more elegant way to do it:

https://github.com/FreeTubeApp/FreeTube/blob/8c3311380bca5a9fda628130a723afbd3145a7a7/src/renderer/components/ft-shaka-video-player/ft-shaka-video-player.js#L1665-L1677